packages/ckeditor5-heading/src/headingconfig.ts

```diff
@@ -74,12 +75,27 @@ export interface HeadingConfig {
  */
 export type HeadingOption = HeadingElementOption | HeadingParagraphOption;
 
+/**
+ * Characters used to distinguish whether a name is a custom model name with a separator.
+ *
+ * @internal
+ */
+type HeadingCustomModelSeparator = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'I' | 'J' | 'K'
+	| 'L' | 'M' | 'N' | 'O' | 'P' | 'Q' | 'R' | 'S' | 'T' | 'U' | 'V' | 'W' | 'X' | '-' | '_' | ':';
+
+/**
+ * Custom name of the model specified by the user.
+ *
+ * @internal
```

Comment:
Why is it internal if it's used in non-internal type (line 98)?

Reply:
@arkflpc It should be exported and public available but we have issue with documentation generation. Umberto incorrectly serializes union in template string literal and in API reference we have something like it:

![obraz](https://github.com/ckeditor/ckeditor5/assets/3949262/eaa77c59-d7b4-4826-a50f-282e9d5e3386)

I tried to split it between types or use it inline directly in `HeadingCustomModel` but it always result in `[object Set]` in docs.